patches/chromium/Clang-build-script-Disable-hwasan.patch

```diff
@@ -0,0 +1,27 @@
+From 122dbe550dbdc2608b4e076193dc4098936e3470 Mon Sep 17 00:00:00 2001
+From: Ryan Gonzalez <rymg19@gmail.com>
+Date: Thu, 30 Sep 2021 11:27:43 -0500
+Subject: [PATCH] Clang build script: Disable hwasan
+
+hwasan doesn't seem to actually compile if not on a host x86-64 system,
```

Comment:
It's...odd for sure, I believe it's a bug in the build scripts but I can't entirely tell. It's basically tied to the x64-only aliasing mode being stuck enabled, which *shouldn't* happen, but no attempts to disable it actually worked. In the end, I went for a patch, just because hwasan isn't actually used on release builds anyway. I do want to find the more proper cause at some point, but without this the aarch64 version won't build.

(This was tied to the M94 update, I just moved it from a `sed` command to an actual patch.)